src/pages/docs/octopus-rest-api/openid-connect/index.md

```diff
@@ -0,0 +1,111 @@
+---
+layout: src/layouts/Default.astro
+pubDate: 2023-09-27
+modDate: 2023-09-27
+title: Using OpenID Connect with the Octopus API
+description: External systems can use OpenID Connect with service accounts to access the Octopus API without needing to provision API keys
+navOrder: 30
+hideInThisSection: true
+---
+
+Octopus supports using [OpenID Connect (OIDC)](https://openid.net/) to access the Octopus API without needing to provision API keys.
+
+Some of the benefits of using OIDC include:
```

Comment:
I'm wondering if we want some sort of preamble to "what is OIDC" including these benefits here.

Reply:
Yeah I agree, It would also be nice to link out to an authoritative site that deals with the topic so that you dont need to go into deep detail here and can defer to the expertise elsewhere.